src/model.js

```diff
@@ -48,12 +60,15 @@ module.exports = function(koop) {
         if(req.url.includes('VectorTileServer')){
             layerId = "0";
             if(req.params.x && req.params.y && req.params.z){
-                const tileBBox = tileIndexToBBox(parseInt(req.params.x), parseInt(req.params.y), parseInt(req.params.z));
-                // logger.debug(`Z: ${req.params.z}, X: ${req.params.x}, Y: ${req.params.y}`);
-                // logger.debug(JSON.stringify(tileBBox));
+                const tileBBox = tileIndexToBBox(parseInt(req.params.x), parseInt(req.params.y), parseInt(req.params.z),
+                    customSymbolizer ? customSymbolizer.tileBuffer : 0);
                 req.query.geometry = tileBBox;
+                req.query.maxAllowableOffset = Math.max(4.864, 0.019 * Math.pow(2, 22 - parseInt(req.params.z)));
```

Comment:
Same as above, move to own function with comments that explain the math and values.

Reply:
Done